packages/suite/src/actions/suite/storageActions.ts

```diff
@@ -12,6 +12,7 @@ import type { GraphData } from '@wallet-types/graph';
 import type { Discovery } from '@wallet-reducers/discoveryReducer';
 import type { FormState } from '@wallet-types/sendForm';
 import type { Trade, TradeType } from '@wallet-types/coinmarketCommonTypes';
+import { FormDraft } from '@wallet-types/formDraft';
```

Comment:
```suggestion
import type { FormDraft } from '@wallet-types/formDraft';
```
lets keep syntax compatibility with above lines

Reply:
Fixed.